packages/serialization/text/test/common/textSerializationWriter.ts

```diff
@@ -0,0 +1,32 @@
+import { assert, describe, it } from "vitest";
+import { TextSerializationWriter } from "../../src";
+
+describe("TextSerializationWriter", () => {
+	it("writeEnumValue", () => {
+		const textSerializationWriter = new TextSerializationWriter();
+
+		const statuses = [LongRunningOperationStatusObject.NotStarted, LongRunningOperationStatusObject.Running];
+		textSerializationWriter.writeEnumValue("", ...statuses);
+		const formContent = textSerializationWriter.getSerializedContent();
+		const form = new TextDecoder().decode(formContent);
+		const expectedString = "notStarted, running";
```

Comment:
why are we expecting a space here?

Reply:
This was a pre-existing format of the output of `writeEnumValue` I can modify the current output to remove the white space

Reply:
please do, I'm not sure why we'd require a space for collections here...